stan/math/fwd/scal/fun/falling_factorial.hpp

```diff
@@ -11,21 +11,7 @@ namespace stan {
 
     template<typename T>
     inline fvar<T>
-    falling_factorial(const fvar<T>& x, const fvar<T>& n) {
-      using boost::math::digamma;
-
-      T falling_fact(falling_factorial(x.val_, n.val_));
-      return fvar<T>(falling_fact,
-                     falling_fact
-                     * (digamma(x.val_ + 1) - digamma(x.val_ - n.val_ + 1))
-                     * x.d_
-                     + falling_fact
-                     * digamma(x.val_ - n.val_ + 1) * n.d_);
-    }
-
-    template<typename T>
-    inline fvar<T>
-    falling_factorial(const fvar<T>& x, double n) {
+    falling_factorial(const fvar<T>& x, int n) {
```

Comment:
There should be doxygen documentation explaining what is being computed and any error conditions and the exceptions thrown.